trough/db_api.py

```diff
@@ -52,11 +52,11 @@ def _do_read(self, query, raw=False):
         self._last_results = results
 
     def _do_write(self, query):
-        """
+"""
```

Comment:
Eight spaces please.  If in doubt, run the code through psf/black.

Reply:
Indentation is very significant in Python unlike many other languages that use {}, [], or () instead of indentation.